nbcelltests/tests/test_test.py

```diff
@@ -722,117 +681,114 @@ def make_test_mod():
 # https://github.com/jpmorganchase/nbcelltests/issues/125 before
 # deciding how to handle that better here.
 
-# runWithReturn
```

Comment:
Please add same comment here as in Makefile (i.e. explain why these tests are grouped into a class, linking to the same github issue if you create the issue).

Reply:
(Sorry, the above comment should have been on the line below, i.e. it's referring to the new `TestCoverage` class.)